src/main/java/redis/clients/jedis/ScanIteration.java

```diff
@@ -1,33 +1,35 @@
 package redis.clients.jedis;
 
+import java.util.Collection;
 import java.util.function.Function;
+
 import redis.clients.jedis.params.ScanParams;
 import redis.clients.jedis.Protocol.Keyword;
 import redis.clients.jedis.providers.ConnectionProvider;
 import redis.clients.jedis.resps.ScanResult;
-import redis.clients.jedis.util.JedisRoundRobinBase;
+import redis.clients.jedis.util.JedisCommandIterationBase;
 
-public class ScanRoundRobin extends JedisRoundRobinBase<ScanResult<String>> {
+public class ScanIteration extends JedisCommandIterationBase<ScanResult<String>, String> {
```

Comment:
"Round Robin" ties it to one technique, but "iteration" implies linearity, so I was thinking something along the lines of "Multiplexer", "Aggregator", "Collector" or even "Scanner"? Add the word "Node" for further specification?